python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -158,7 +159,9 @@ def compile(model: nn.Module,
             except ImportError:
                 raise RuntimeError("You should install onnx and onnxruntime to set `onnx=True`, "
                                    "or just set `onnx=False`.")
-
+        elif openvino:
```

Comment:
I think we should check the correct combinations of onnx, openvino and quantize - some of them cannot be true at the same time.

Reply:
onnx and openvino can't be True at the same time. Add error here in this case.

Reply:
Can openvino and quantize both be ture?

Reply:
In the future, I think yes. Right now we don't support this. Add another error for this? Shall I set quantize as False in this PR?  
Otherwise, uses have to use `Trainer.compile(model,  quantize=False, openvino=True)`.

Reply:
Yes - set quantize to false.